modules/insights-operator-configuring.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-operator.adoc
+
+
+:_content-type: PROCEDURE
+[id="insights-operator-configuring-sca_{context}"]
+= Configuring Insights Operator
+
+You can configure Insights Operator to meet the needs of your organization. The Insights Operator is configured using a combination of the default configurations in the `config/pod.yaml` file and configurations stored in the `support` secret in the `openshift-config` namespace. The `support` secret does not exist by default and must be created when adding custom configurations for the first time. Configurations in the `support` secret will override the defaults set in the `config/pod.yaml` file.
+
+The table below describes the available configuration attributes:
+
+.Insights Operator configurable attributes
+[options="header"]
+|====
+|Attribute name|Description|Value type|Default value
+|`username`|Username for basic authentication with `console.redhat.com`(overrides the default `pull-secret` token authentication when set)|String|Not set
+|`password`|Password for basic authentication with `console.redhat.com`(overrides the default `pull-secret` token authentication when set)|String|Not set
+|`enableGlobalObfuscation`|Enables the global obfuscation of IP addresses and the cluster domain name|Boolean|`false`
+|`scaInterval`|Frequency of the simple content access entitlements download|Time interval|`8h`
+|`scaPullDisabled`|Disables the simple content access entitlements download|Boolean|`false`
+|`clusterTransferInterval`|How often Insights Operator checks OpenShift Cluster Manager for available cluster transfers|Time interval|`24h`
```

Comment:
Hmm I think I would rather not mention the two intervals here. I am afraid someone will set 1 minute there and then forget to remove or change and the cluster will be ddosing the API.

Reply:
The only reason I kept these ones is that we mention using time intervals in the procedures for SCA and Cluster transfers. I can remove them here, but they will still be included in those procedures: 
https://docs.openshift.com/container-platform/4.12/support/remote_health_monitoring/insights-operator-simple-access.html#insights-operator-configuring-sca_remote-health-reporting-from-restricted-network

Reply:
OK